mount_config.go

```diff
@@ -142,6 +142,8 @@ type MountConfig struct {
 	// For expert use only! May invalidate other guarantees made in the
 	// documentation for this package.
 	Options map[string]string
+
+	SubType string
```

Comment:
Please add documentation. Something like:

    // The subtype to report for the file system, reflected in e.g. mount(8), which will report the type as "fuse.<subtype>".